notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb

```diff
@@ -0,0 +1,1283 @@
+{
```

Comment:
<p>remove?</p><p>'''</p><p>....</p><p>'''</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>This is the code for the prediction container</p>